### DISCUSSION TITLE
The type of this node cannot be serialized because its property '[$$PropertyValue]' cannot be serialized.

### DISCUSSION THREAD

**Discussion by jackguoAtJogg** [stitchesjs/stitches#1081] [Help]
I'm not sure how many people have this issue experience. I'm on Typescript 4.5.2 and getting the same error, and the error came out from no where.

https://github.com/stitchesjs/stitches/issues/1078

**Comment by ygorluiz** [comment]
this can help you https://github.com/stitchesjs/stitches/pull/1082#issue-1362651408
